### ISSUE TITLE
Filter panel: Search close button needs an accessible name

### ISSUE THREAD

**Issue by em843** [flexmonster/ngx-flexmonster#1]
![Image](https://github.com/user-attachments/assets/b79253e7-1268-43b4-8291-07efe42c47d7)

The button is missing an accessible name/label. As a result its functionality or purpose is not conveyed to screen reader users.
Element description: 
- Close button in the 'Search' field.

**Comment by se7enteenblue** [comment]
Hello Erin,

Thank you for reporting the issue.

Please find our answer in the following thread: https://www.flexmonster.com/question/raising-accessibility-issues-with-flexmonster/ 

Kind regards,
Nadia
